front/js/components/category.es6

```diff
@@ -127,6 +127,8 @@
   /**
    * Load Products from server.
    * Publish 'onProductLoad' event on success.
+   *
+   * @todo #302:60m Implement pagination logic for frontend.
```

Comment:
The most important comment.
Let's clarify about nested pdd issues.
Currently have this ones:

- front/js/components/category.es6
`@todo #302:60m Implement pagination logic for frontend.`

- shopelectro/urls.py
`@todo #302:60m Implement pagination ability for load_more urls.`

- templates/catalog/category_products.html
`{# @todo #302:60m Implement pagination buttons. #}`

There is not clear distinction between this tasks.
- pagination ability may be written with js. So, it can be part of the first task
- pagination buttons is too hardly coupled with first task.
Because we should put ready buttons in js logic.

In this hard-coupled tasks situation we should create only one task.
For example:
```
@todo #302:60m Render pagination for seo.
 - Render pagination buttons and hide them with js code.
 - Load_more button should reniew canonical url for current page.
```

And implementor will split this task in his own.
This splitting will be more clear then current one.
Because he'll leave ready some parts of this task.